finesse/hardware/device.py

```diff
@@ -76,19 +76,26 @@ class AbstractDevice(ABC):
 
     The key represents the parameter name and the value is a list of possible values.
     """
+    _device_async_open: ClassVar[bool | None] = None
+    """Whether the device opens asynchronously (i.e. completes after __init__)."""
 
     def __init_subclass__(
-        cls, parameters: Mapping[str, str | tuple[str, Sequence]] = {}
+        cls,
+        parameters: Mapping[str, str | tuple[str, Sequence]] = {},
+        async_open: bool | None = None,
```

Comment:
What's the reason of being `None`? Should be either async or not?

Reply:
If it's `None`, then it just leaves the `_device_async_open` property as whatever it was before, so if it was set by a parent class, it won't be overridden.